tests/data/rnx-updu_rn3214.json

```diff
@@ -0,0 +1,23 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.55108.1.0",
+                    "sysDescr": "RNX UPDU RN3214 (UPDU-Right), S/N: 100124, Version 2.7.0-5741e7a3",
+                    "sysContact": "<private>",
+                    "version": "2.7.0-5741e7a3",
+                    "hardware": "RN3214",
+                    "features": null,
+                    "location": "<private>",
+                    "os": "rnx-updu",
+                    "type": "power",
+                    "serial": "100124",
+                    "icon": "riedo.png"
+                }
+            ]
+        },
+        "poller": "matches discovery"
+    }
+}
```

Comment:
Hmmm something is wrong since this file doesn't contain the sensors discovered. Try deleting it and regenerate it

Reply:
Yeah, during testing of the os paramters I ran the script with -m os and forgot to do a full run once everything was wokring :man_facepalming: 

Sorry